tools/main.cpp

```diff
@@ -32,6 +33,15 @@ int main(int argc, char** argv)
     auto clearPsuFaultLed = app.add_flag("-c, --clearPsuFaultLed",
                                          "Clears PSU fault LEDs.");
 
+    std::string objectPath;
+    auto objectPathOption = app.add_option("-O, --object", objectPath,
+                                           "Object path of FRU.");
+    auto syncFaultLedFlag =
+        app.add_flag(
+               "-q, --syncFaultLed",
+               "Syncs asserted and physical properties of fault LED of a FRU according to the functional property of the FRU.")
```

Comment:
"Sync fault LEDs to functional state of the FRU"
Lets keep the info limited and to the point.